### DISCUSSION TITLE
Are new INI options auto-added?

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1423] [Q&A] [answered]
If a new setting is added to the INI, such as `AutoLoadScripts` in [commit `5a8cdce`](https://github.com/scripthookvdotnet/scripthookvdotnet/commit/5a8cdce8b1fac643e9ca494ea7aadc29e1ac63e4), will the option generate automatically after first run of SHVDN or does the setting need to be added there by the user (either manually or by installing the new INI)?

**Comment by Michael21107** [comment]
@kagikn

**Reply by kagikn** [reply]
I feel the pain, but could you be more patient until v3.7.0 is released? Since #1370 is also worth trying as well as this one but difficult for me to solve.

**Reply by Michael21107** [reply]
its a question about how it is currently, not a suggestion for creating/updating a feature

**Reply by kagikn** [reply, marked as answer]
sorry i thought wrong, currently you have to add new options and set new values manually if you want to change them to anything other than the defaults

**Reply by Michael21107** [reply]
gotcha, thanks
